src/cc_helpers.cc

```diff
@@ -117,6 +117,15 @@ void certifier::framework::cc_trust_data::cc_trust_data_default_init() {
 }
 
 certifier::framework::cc_trust_data::~cc_trust_data() {
+  for (int i = 0; i < num_certified_domains_; i++) {
```

Comment:
@jlmucb -- Restored the correct delete / release-memory code in the desctructor.

See my notes in the test case.